jwst/associations/lib/rules_level3.py

```diff
@@ -1152,7 +1152,7 @@ def __init__(self, *args, **kwargs):
             DMSAttrConstraint(
                 name='exp_type',
                 sources=['exp_type'],
-                value='nis_wfss',
+                value='nis_wfss|nis_extcal',
```

Comment:
I think `|nis_extcal` needs to be removed here too, and the SimpleConstraint needs to be moved up here and wrapped in another Constraint.   The logic should be either exp_type = nis_wfss, or else valid extcal.

Reply:
good catch, thank you